crates/neon/Cargo.toml

```diff
@@ -21,7 +21,7 @@ nodejs-sys = "0.13.0"
 
 [dependencies]
 libloading = "0.7.3"
-semver = "0.9.0"
+semver = "1"
```

Comment:
This is a breaking change since `semver::Version` is exposed in Neon's public API, but we already have some breaking changes staged and this seemed pretty minor.

We may want to consider completely removing the `meta` feature prior to 1.0 since users could leverage the [auditable](https://crates.io/crates/auditable) crate for this.

Reply:
That's a good point, and just the fact that we're exporting public API that uses types from another crate (`semver`) creates some risk.